### PR TITLE
 [MIG] partner_payment_return_risk: Migration to 10.0

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa

### CODE REVIEW

File: partner_payment_return_risk/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2018 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': 'Partner Payment Return Risk',
+    'version': '10.0.1.0.0',
```
Comment (rafaelbn) [inline partner_payment_return_risk/__manifest__.py:6]: Is intentional to exclude Category and website?
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L9

**Review by rafaelbn — COMMENTED** [review]
Thanks @carlosdauden . Travis and runbot are :red_circle:

**Review by chienandalu — COMMENTED** [review]
Seems that you need `account-payment` in oca_dependencies

**Comment by rafaelbn** [comment]
@carlosdauden are you going to finish this PR? It should be very nice but if not just close it

**Review by rafaelbn — APPROVED** [review]
This was in production for long time

**Comment by pedrobaeza** [comment]
Other that has been deprecated due to migrated databases to upper versions, so closing. If someone wants to resume, can take this PR as base.

**Pull request closed without merging**